### PR TITLE
Update company-portal.md

### PR DESCRIPTION
**Pull request by mertenar**
adding link for this to help admins @jaimeo

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/get-started/company-portal.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ For more information on submitting support requests with Microsoft Managed Deskt
 ## Step 3 - Communicate change to your users
 As the IT administrator for your organization, it’s important to let your users know how to use Company Portal in your organization. Microsoft Managed Desktop recommends:
 - Steps on installing applications from the Company Portal. For more information, see [Install and share apps on your device](https://docs.microsoft.com/intune-user-help/install-apps-cpapp-windows).
-- How to send requests to IT administrators for applications that are not currently available.
+- How to send requests to IT administrators for applications that are not currently available. Documentation on this can be found [here in the Intune user help documentation](https://docs.microsoft.com/intune-user-help/install-apps-cpapp-windows#request-an-app-for-work-or-school).
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/company-portal.md]:
```suggestion
- How to send requests to IT administrators for applications that are not currently available. For more information, see [Request an app for work or school](https://docs.microsoft.com/intune-user-help/install-apps-cpapp-windows#request-an-app-for-work-or-school).  
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Comment by mertenar** [comment]
@jaimeo incorporated

**Pull request merged**